### ISSUE TITLE
home and end key not working at all for me

### ISSUE THREAD

**Issue by menash134** [Eugeny/terminus#4491]
Hi,
I use serial connections and ssh to my linux machine. Home and End keys are getting overriden somewhere and I could not use it. 
 I saw the hotkey mappings, but home and end keys are not mapped at all. What is happening to those keys? Could you help?

**Comment by nibble-4bits** [comment]
Yeah, it's not working for me either, and I'm primarily using Git Bash as shell. 

For instance, when I want to reach the end of a file with `less` using the End key or the start of the file with the Home key, nothing happens.

My environment:
Version: 1.0.155
Platform: Windows x64 10.0.19043
Plugins: none
Frontend: xterm

**Comment by manojkiraneda** [comment]
I have the same issue as well

**Comment by Vimos** [comment]
Same issue with version `1.0.181` on `Windows 22H2 22621.232`.

**Comment by sstudi** [comment]
Same issue:

Version: 1.0.204
Platform: Windows x64 10.0.22621
Plugins: fit-screen
Frontend: xterm-webgl

**Comment by sstudi** [comment]
Got it working by removing `end` and `home` as hotkey, as described in #4874.
